samples/cpp/speculative_decoding_lm/speculative_decoding_lm.cpp

```diff
@@ -40,9 +40,17 @@ int main(int argc, char* argv[]) try {
     scheduler_config.cache_size = 5;
     scheduler_config.block_size = get_default_block_size(main_device);
 
+    ov::genai::SchedulerConfig draft_scheduler_config;
+    draft_scheduler_config.cache_size = 5;
```

Comment:
do we need to pass cache_size for draft model?
the idea was to split cache between main and draft model 

CC @iefode